readme_template.md

```diff
@@ -0,0 +1,97 @@
+
+<!-- TOC depthfrom:2 -->
+
+- [:whale: or :arrow_forward: Run a release](#whale-or-arrow_forward-run-a-release)
+- [:construction: or :hammer_and_wrench: Prerequisites or Setup](#construction-or-hammer_and_wrench-prerequisites-or-setup)
+- [:desktop_computer: Development](#desktop_computer-development)
+    - [Getting Started](#getting-started)
+    - [Building](#building)
+    - [Testing](#testing)
+- [:handshake: Contributing](#handshake-contributing)
+- [:rocket: Production](#rocket-production)
+- [:art: Assets or Sprites](#art-assets-or-sprites)
+- [:blue_book: Application related](#blue_book-application-related)
+- [:pager: CLI related](#pager-cli-related)
+- [:whale: Docker related](#whale-docker-related)
+- [:scroll: Guidelines](#scroll-guidelines)
+- [:arrows_counterclockwise: Sync](#arrows_counterclockwise-sync)
+- [:mag: Links to further information](#mag-links-to-further-information)
+
+<!-- /TOC -->
+
+## :whale: or :arrow_forward: Run a release
+
+If running a release is the easiest way to run the application or services of the repository, this section should be the first one.
+
+Describe where the release can be downloaded (`docker pull` or release download) and how to start it locally. This must include all required environment variables. Use the :whale: icon for docker containers and the :arrow_forward: for all other releases.
+
+If there is a lot of configuration, provide the command to start the service with our scripts in [docker-images](https://github.com/skillslab/docker-images) first, otherwise after the [Docker](https://www.docker.com/) or [Docker Compose](https://docs.docker.com/compose/) command.
+
+## :construction: or :hammer_and_wrench: Prerequisites or Setup
+
+You have to add clone instructions if a simple pull is not sufficient. For example if submodules have to be checked out.
+
+Add instructions for software that has to be installed on the machine to run the code or link to another repository or other documentation which covers the setup. Ofter used examples are [ansible repo](https://github.com/skillslab/ansible) and [Dev Containers](https://code.visualstudio.com/docs/devcontainers/containers).
+
+## :desktop_computer: Development
+
+### Getting Started
+
+Describe how to start up the development environment. Either with docker, Visual Studio Code and Dev Containers or the Unreal Engine. If there are multiple ways, mention all, but start with the most used. Also mention the differences.
+
+### Building
+
+In this section describe how the projection can be built and describe which configurations are available.
+
+### Testing
+
+Explain where the tests are organized and how they can be executed.
+
+## :handshake: Contributing
+
+**TODO:** Merge with guidelines?
+
```

Comment:
can we add a section on **Documentation** ? link to where it is, how it is built (e.g., with Sphinx, CodeTour, Pyreverse).

can we add a section on **Logging**?

Reply:
I would but the link to auto generated documentation under **Links to further information**.
What would be in the Logging section? Where the log files can be found?

Reply:
yes, and the type of logger

Reply:
This should now be in the `CONTRIBUTING.md` file